components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/main/java/org/wso2/carbon/apimgt/rest/api/core/impl/ApisApiServiceImpl.java

```diff
@@ -58,4 +63,39 @@ public Response apisApiIdGatewayConfigGet(String apiId, String accept,
                     MediaType.APPLICATION_JSON).entity(errorDTO).build();
         }
     }
+
+    /**
+     *
```

Comment:
Since the parent class is an auto generated class from swagger generation we add Java doc in Impl class. 
Missing method description is added by https://github.com/wso2/carbon-apimgt/pull/4232/commits/1435317770001cf742c3b50f56ee31316a078b85